product_subscription/models/invoice.py

```diff
@@ -39,18 +39,25 @@ def get_next_start_date(self, subscriber):
 
     @api.multi
     def process_subscription(self, effective_date):
+        # todo should be delegated subscription request
```

Comment:
to subscription request ?

Reply:
Yes, I think i't d be cleaner to do this computation in the subscription.request model.

Reply:
the it should be " # todo should be delegated **to** subscription request"

Reply:
ah, ok 👌